src/App.css

```diff
@@ -1,72 +1,224 @@
-  }
-  /* end style for register section  */
-.form-control{
-  background-color: transparent !important;
-  border: none !important;
-  color: white !important;
-  padding: 0.5rem  0 !important;
-  fill: none;
+/* register style */
+
+.bg-reg {
+  background-image: url("/src/assets/images/bg-register.svg");
+}
+.bg-login {
+  background-image: url("/src/assets/images/bg-login.svg");
+}
+.bg-forget {
+  background-image: url("/src/assets/images/bg-forget.svg");
+}
+.bg-change {
+  background-image: url("/src/assets/images/bg-change.svg");
+}
+.bg-reset {
+  background-image: url("/src/assets/images/bg-reset.svg");
+}
+.bg-verify {
+  background-image: url("/src/assets/images/bg-verify.svg");
+}
+.form {
+  background-color: rgba(49, 89, 81, 0.9);
+}
+.bg-main {
+  background: #ef9b28 !important;
+}
+.text-main {
+  color: #ef9b28 !important;
+}
+.container-auth-form {
+  width: 90%;
+  margin: 0 auto;
+}
+.form-title h3 {
+  font-family: Inter;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 46px;
+  letter-spacing: -0.4000000059604645px;
+  text-align: left;
+  position: relative;
+}
+
+.form-title p {
+  position: absolute;
+  bottom: 20%;
+  font-family: Inter;
+  font-size: 13px;
+  font-weight: 500;
+  line-height: 46px;
+  letter-spacing: -0.4000000059604645px;
+  text-align: left;
+  color: #fff;
+}
+.form-title h3::after {
+  content: "";
+  position: absolute;
+  background: #ef9b28;
+  width: 16px;
+  height: 3.57px;
+  top: 82%;
+  right: 97%;
+}
+
+.input-container {
+  position: relative;
+  margin: 35px 0;
+}
+
+.input-field {
+  font-family: Inter;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 22px;
+  letter-spacing: 0.01em;
+  display: block;
+  width: 100%;
+  padding: 10px;
   border: none;
-  border-bottom: 1px solid #ffffff;
+  border-bottom: 1px solid #ccc;
+  outline: none;
+  background-color: transparent;
+  color: #fff;
+}
+
+.input-field::placeholder {
+  color: #ccc;
+}
+
+.input-label {
+  position: absolute;
+  top: 0;
+  left: 0;
   font-size: 16px;
-} 
-.form-control::placeholder{
-  color: white !important;
-  padding-left: -10px !important;
+  color: rgba(204, 204, 204, 0);
+  pointer-events: none;
+  transition: all 0.3s ease;
+}
 
+.input-highlight {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  height: 2px;
+  width: 0;
+  background-color: #ef9b28;
+  transition: all 0.3s ease;
 }
-.form-control::focus {
-  border: none !important;
+
+.input-field:focus + .input-label {
+  font-family: Inter;
+  font-weight: 400;
+  line-height: 22px;
+  letter-spacing: 0.01em;
+  top: -20px;
+  font-size: 16px;
+  color: #ef9b28;
 }
-.border_bottom{
+
+.input-field:focus + .input-label + .input-highlight {
   width: 100%;
-  border-bottom: .5px solid rgba(206, 205, 205, 0.507);
 }
 
-/*custom color for button  */
-.color-button {
-  background-color: #EF9B28 !important;
-  color: #ffffff;
-  font-size: 16px;
+/* main btn */
+.main-btn {
+  background-color: #ef9b28;
```

Comment:
Refer back to comments  in this pull request and why we should be using css variables instead of fixed colors.
https://github.com/OmarBazeed/project-mangement-system/pull/6